op-dispute-mon/flags/flags.go

```diff
@@ -28,22 +28,22 @@ var (
 		Usage:   "HTTP provider URL for L1.",
 		EnvVars: prefixEnvVars("L1_ETH_RPC"),
 	}
-	FactoryAddressFlag = &cli.StringFlag{
+	GameFactoryAddressFlag = &cli.StringFlag{
```

Comment:
Not sure why we renamed this to not match the actual CLI flag name.

Reply:
I updated this to match the cli flag name though?